src/dsp_tools/utils/xml_utils.py

```diff
@@ -66,6 +67,22 @@ def remove_qnames_and_transform_special_tags(input_tree: etree._Element) -> etre
     return input_tree
 
 
+def transform_into_qnames(root: etree._Element) -> etree._Element:
```

Comment:
I dislike the code redundancy that this introduces. In fact, it is a good idea to have this new function. But now it should be integrated in the existing code. This means, the qname transformation should be removed from the function`remove_qnames_and_transform_special_tags()`, and instead, your new function should be used.